src/main/java/org/gridsuite/study/server/service/shortcircuit/ShortCircuitService.java

```diff
@@ -229,31 +230,61 @@ public static ShortCircuitParametersEntity toEntity(ShortCircuitParameters param
                 parameters.isWithFortescueResult(),
                 parameters.isWithFeederResult(),
                 parameters.getStudyType(),
-                parameters.getMinVoltageDropProportionalThreshold());
+                parameters.getMinVoltageDropProportionalThreshold(),
```

Comment:
I think you have a bug when you call this method in insertDuplicatedStudy().
The toEntity method should not ignore one parameter, it's a source of bug

Reply:
> I think you have a bug when you call this method in insertDuplicatedStudy(). 

what type of bug because I tested the study duplication and it works fine.
In order to create ShortCircuitParametersEntity I use the parameters values except the predefinedParameters. So I use ShortCircuitPredefinedConfiguration.ICC_MAX_WITH_NOMINAL_VOLTAGE_MAP because it's the default value

Reply:
When we duplicate a study, we duplicate its shortcircuit parameters. We should not use the default value but the value selected in the source study. There's no reason to exclude the predefinedParameters nope ?

Reply:
Indeed for insertDuplicatedStudy we need the predefinedParameters of the source study. 
Fixed.